.travis.yml

```diff
@@ -16,5 +16,5 @@ install:
   - export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
 
 script:
-  - make check
```

Comment:
Ok, but make check still runs with no error, but no check.
Can we alias?
Should we modify the doc, I do not see this modification in the list.